### ISSUE TITLE
Implementing MultiSig 

### ISSUE THREAD

**Issue by vaibhavchellani** [merkaliser/scilla-vanilla#28]
Multisig wallet contract would be a great addition to this repo .

**Comment by AmritKumar** [comment]
@vaibhavchellani You may want to skip this. One of the core Scilla team members has already implemented a multisig wallet contract: https://github.com/Zilliqa/scilla/tree/61_payment_channel/tests/contracts/wallet

**Comment by vaibhavchellani** [comment]
yes i shared it our internal channel that there is a multisig implementation made . Thanks ! Can you think of any variation of multisig we could implement ? For example we could do multisig for fungible/non-fungible tokens ?

**Comment by AmritKumar** [comment]
@vaibhavchellani Yes, Multisig for fungible tokens will be a good idea.

**Comment by merkaliser** [comment]
@abhishektanwar any updates ?
